gen/rules/src/main/scala/clue/gen/annotations.scala

```diff
@@ -36,6 +31,6 @@ class AnnotationPattern[T <: StaticAnnotation](implicit classTag: ClassTag[T]) {
   }.flatten
 }
 
-object GraphQLSchemaAnnotation extends AnnotationPattern[GraphQLSchema]
+object GraphQLSchemaAnnotation extends AnnotationPattern("clue.annotation.GraphQLSchema")
 
-object GraphQLAnnotation extends AnnotationPattern[GraphQL]
+object GraphQLAnnotation extends AnnotationPattern("clue.annotation.GraphQL")
```

Comment:
By hard-coding the strings instead of relying on the classes, we no longer need to depend on clue itself in the rules. This helps avoid problems that clue is cross-build for 2.13/3 but the rules are only on 2.13.